### ISSUE TITLE
Automatic focus into search field after opening

### ISSUE THREAD

**Issue by mmokross** [BMSVieira/BVSelect-VanillaJS#2]
Additional to #1: If the dropdown is opened using mouse click and the search field has been enabled, it the keyboard focus should be set into the search field immediately?

**Comment by mmokross** [comment]
👍
